### PR TITLE
Implemented sentiment overview with OpenAI

### PR DESCRIPTION
**Pull request by npudhota**
Addressed comments from previous pull request

### CODE REVIEW

File: lib/presentation/widgets/sentiment_details.dart [hunk 1]
```diff
@@ -99,7 +99,7 @@ class _SentimentCard extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return Container(
-      height: 500,
+      height: 455,
```
Comment (Thouny) [inline lib/presentation/widgets/sentiment_details.dart]: Please revert this change. It causing a bottom overflow

Comment (npudhota) [reply lib/presentation/widgets/sentiment_details.dart]: Addressed in commit 03882b2c199e1b66065f7db3e2d6295b41e04137

**Review by Thouny — APPROVED** [review]
LGTM 👍
